plugins/outputs/cloudwatchlogs/pusher_test.go

```diff
@@ -44,10 +45,16 @@ func (s *svcMock) CreateLogStream(in *cloudwatchlogs.CreateLogStreamInput) (*clo
 	}
 	return nil, nil
 }
+func (s *svcMock) PutRetentionPolicy(in *cloudwatchlogs.PutRetentionPolicyInput) (*cloudwatchlogs.PutRetentionPolicyOutput, error) {
```

Comment:
Should probably add a test for the put retention policy aside from just setting up the mock.